refarch-integrations/refarch-email-integration/refarch-email-integration-core/src/main/java/de/muenchen/refarch/email/integration/domain/model/TextMail.java

```diff
@@ -0,0 +1,31 @@
+package de.muenchen.refarch.email.integration.domain.model;
+
+import jakarta.validation.constraints.NotBlank;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * Object contains all the information needed to send a mail.
+ */
+@EqualsAndHashCode(callSuper = true)
+@Data
+@RequiredArgsConstructor
```

Comment:
```suggestion

```

`@Data` already contains `RequiredArgsConstructor`

Reply:
For some reason that breaks the build, but i made the classes unmodifiable and removed the `@Data` annotation.